set.mm

```diff
@@ -509857,6 +509867,251 @@ real number multiplication operation (this has to be defined in the main
+         (Contributed by Thierry Arnoux, 3-Aug-2025.) $)
+      mndractf1 $p |- ( ph -> G : B -1-1-> B ) $=
+        ( wceq co wcel vi vj wf cv cfv wi wral wf1 wa adantr simpr mndcld fmptd
+        cvv oveq1 simpllr ovexd fvmptd3 simplr 3eqtr3d oveq1d ad3antrrr mndassd
+        cmnd oveq2d mndrid syl2anc ex anasss ralrimivva dff13 sylanbrc ) ABBEUC
+        UAUDZEUEZUBUDZEUEZRZVMVORZUFZUBBUGUABUGBBEUHAIBIUDZFCSZBEAVTBTZUIBCDVTF
+        JLADVDTZWBNUJAWBUKAFBTZWBOUJULMUMAVSUAUBBBAVMBTZVOBTZVSAWEUIZWFUIZVQVRW
+        HVQUIZVMHCSZVOHCSZVMVOWIVMFGCSZCSZVOWLCSZWJWKWIVMFCSZGCSVOFCSZGCSWMWNWI
+        WOWPGCWIVNVPWOWPWHVQUKWIIVMWAWOBEUNMVTVMFCUOAWEWFVQUPZWIVMFCUQURWIIVOWA
+        WPBEUNMVTVOFCUOWGWFVQUSZWIVOFCUQURUTVAWIBCDVMFGJLAWCWEWFVQNVBZWQAWDWEWF
+        VQOVBZAGBTWEWFVQPVBZVCWIBCDVOFGJLWSWRWTXAVCUTWIWLHVMCAWLHRWEWFVQQVBZVEW
+        IWLHVOCXBVEUTWIWCWEWJVMRWSWQBCDVMHJLKVFVGWIWCWFWKVORWSWRBCDVOHJLKVFVGUT
+        VHVIVJUAUBBBEVKVL $.
+    $}
+
+    $d .+ a z $.  $d .+ x $.  $d .0. a y z $.  $d .0. x $.  $d B a z $.
+    $d B x y z $.  $d G a z $.  $d G x y z $.  $d X a z $.  $d X x $.
+    $d a ph z $.  $d ph x y z $.
+    $( An element ` X ` of a monoid ` E ` is right-invertible iff its
+       right-translation ` G ` is surjective.  (Contributed by Thierry Arnoux,
+       3-Aug-2025.) $)
+    mndractfo $p |- ( ph -> ( G : B -onto-> B <-> E. y e. B ( y .+ X ) = .0. )
+       ) $=
+      ( vz co wceq wa wcel vx wfo cv wrex cfv simpr mndidcl syl adantr foelcdmi
+      cmnd syl2anc cvv oveq1 ovexd fvmptd3 eqeq1d biimpd reximdva mpd wf mndcld
+      wral fmptd ad2antrr eqeq2d ad3antrrr simpllr mndassd simplr oveq2d mndrid
+      fveq2 eqtr2d 3eqtr4rd rspcedvdw ralrimiva dffo3 sylanbrc r19.29an impbida
+      ) ACCFUBZBUCZGDQZHRZBCUDZAWBSZWCFUEZHRZBCUDZWFWGWBHCTZWJAWBUFAWKWBAEUKTZW
+      KNCEHJKUGUHUIBCCFHUJULWGWIWEBCWGWCCTZSZWIWEWNWHWDHWNIWCIUCZGDQZWDCFUMMWOW
+      CGDUNWGWMUFWNWCGDUOUPUQURUSUTAWEWBBCAWMSZWESZCCFVAZPUCZUAUCZFUEZRZUACUDZP
+      CVCWBAWSWMWEAICWPCFAWOCTZSCDEWOGJLAWLXENUIAXEUFAGCTZXEOUIVBMVDVEWRXDPCWRW
+      TCTZSZXCWTWTWCDQZFUEZRUAXICXAXIRXBXJWTXAXIFVMVFXHCDEWTWCJLAWLWMWEXGNVGZWR
+      XGUFZAWMWEXGVHZVBZXHXIGDQZWTWDDQZXJWTXHCDEWTWCGJLXKXLXMAXFWMWEXGOVGVIXHIX
+      IWPXOCFUMMWOXIGDUNXNXHXIGDUOUPXHXPWTHDQZWTXHWDHWTDWQWEXGVJVKXHWLXGXQWTRXK
+      XLCDEWTHJLKVLULVNVOVPVQUAPCCFVRVSVTWA $.
+  $}
+
+  ${
+    $d .+ a u v y $.  $d .0. a u v y $.  $d B a u v y $.  $d F a u v y $.
+    $d X a u v y $.  $d a ph u v y $.
+    mndlactf1o.b $e |- B = ( Base ` E ) $.
+    mndlactf1o.z $e |- .0. = ( 0g ` E ) $.
+    mndlactf1o.p $e |- .+ = ( +g ` E ) $.
+    mndlactf1o.f $e |- F = ( a e. B |-> ( X .+ a ) ) $.
+    mndlactf1o.e $e |- ( ph -> E e. Mnd ) $.
+    mndlactf1o.x $e |- ( ph -> X e. B ) $.
+    $( An element ` X ` of a monoid ` E ` is invertible iff its
+       left-translation ` F ` is bijective.  See also ~ grplactf1o .  Remark in
+       chapter I. p. 17 .  (Contributed by Thierry Arnoux, 3-Aug-2025.) $)
```

Comment:
"Remark in chapter I. p. 17 ." -> "Remark in chapter I. of [BourbakiAlg1] p. 17."

Reply:
Fixed in https://github.com/metamath/set.mm/pull/4961/commits/a5d654e35150fdf413e9fb56d398bb202d0ae33c